### PR TITLE
Adds a summary blog post about Grace Hopper 2015

### PR DESCRIPTION
**Pull request by annegentle**
I have permission from the Rackers in the photo to post it.

### CODE REVIEW

**Comment by sarasafavi** [comment]
Thanks again for running this by us @annegentle -- LGTM!

**Comment by eglute** [comment]
Thanks Anne, looks good to me!

File: _posts/2015-10-21-rackspace-grace-hopper.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+categories:
+- events
+- Grace Hopper
+- outreach
+- women in technology
+- inclusion
+authorIsRacker: true
+---
+
+The woman on her cell phone in the Houston Conference Center hallway laughed lightly and said, “I’m at Grace Hopper. Yes, Grace Hopper. It’s a celebration of women in computing! Yes, Houston. Yes, 12,000 women in technology!” I would have loved to hear what I guess was incredulity on the other side of that conversation. Yes, there are well more than 12,000 women in technology, and we converged in Houston last week.
```
Comment (catlook) [inline _posts/2015-10-21-rackspace-grace-hopper.md]: ... are well over 12,000 ...   [ed note: or maybe "way more than"]

File: _posts/2015-10-21-rackspace-grace-hopper.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+<!-- more -->
+
+I’ve been to the Grace Hopper conference three times now. In Minnesota there were over [4,000 people in 2013](http://gracehopper.anitaborg.org/2013/). Last year in Arizona there were 8,000 people, and this year in Houston we topped 12.000 people, and believe me, it is mostly women. All the [talks have been uploaded](https://www.dropbox.com/sh/1bzkn5apo2uife1/AABodG7ksbFJfb-d9JMADrs3a?dl=0) and there were over 500 talks in seven clusters: Career, Emerging Tech/Best of, General Sessions, Open Source, Organizational Transformation, Productization, and Technology.
```
Comment (catlook) [inline _posts/2015-10-21-rackspace-grace-hopper.md]: ... In Minnesota, there were ...
... Last year, in Arizona, there were ...
... this year, in Houston, we topped 12.000 people. Believe me, it ...
... All the [talks have been uploaded](https://www.dropbox.com/sh/1bzkn5apo2uife1/AABodG7ksbFJfb-d9JMADrs3a?dl=0), and there were ...

File: _posts/2015-10-21-rackspace-grace-hopper.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
... Open Source Documentation](http://www.slideshare.net/annegentle/git-and-github-for-open-source-documentation). First I listened intently to Elizabeth Joseph’s [talk about the system admin work she does for the OpenStack Infrastructure project](https://www.dropbox.com/sh/1bzkn5apo2uife1/AACjMUNfo_gYi7qVGWKZPY8ia/Elizabeth%20K.%20Joseph/Thursday/R1430_sess256_Elizabeth_Joseph.pdf?dl=0). Afterwards I had the sweetest experience, meeting Marina Zhurakhinskaya, the coordinator for the Outreach program for Women. She has done more for women in open source than anyone I know, tackling the issue she saw of fewer women applying for open source internships than men. I gave her a huge hug! She recently received an [O’Reilly Open Source Award](http://opensource.com/life/15/8/interview-marina-zhurakhinskaya-outreachy). She makes me smile so, so much. She’s busy right now as the next round of internship applicants are busy completing their small patch, finding mentors, and identifying projects they can work on for three months in open source. The deadline is November 2nd, and you can read more about how to apply for [OpenStack’s Outreachy internships](https://wiki.openstack.org/wiki/Outreachy).
```
Comment (catlook) [inline _posts/2015-10-21-rackspace-grace-hopper.md]: ... I also gave my talk: [Collaborating on ...
...  Afterwards, I had he sweetest experience meeting...
... right now, as the next ....

File: _posts/2015-10-21-rackspace-grace-hopper.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Speaking of open source, Egle Sigler and Dana Bauer led a workshop on Open Source Day teaching us how to use OpenStack resources to launch an entire application. The [materials are all available on GitHub](https://github.com/danabauer/app-on-openstack) and Egle said it was the highlight of the conference for her.
```
Comment (catlook) [inline _posts/2015-10-21-rackspace-grace-hopper.md]: ... Open Source Day, teaching us ...
... /app-on-openstack), and Egle said it ...

File: _posts/2015-10-21-rackspace-grace-hopper.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+I asked more Rackers for their favorite memory of the conference. I personally love this [photo in the car of a Racker Tools team driving from San Antonio to Houston](https://twitter.com/sarasomewhere/status/653984386128330752).
+
+![Racker Road Trip to Grace Hopper]({% asset_path 2015-10-21-rackspace-grace-hopper/ghc2015roadtrip.jpg %})
+
+When I asked Sara Safavi about her favorite part of Grace Hopper, she said it was the Building Your Career workshop. She and Kacie Houser got to meet and mentor more than 80 students in less than three hours. I saw them in the hallway after their session and they both exclaimed how awesome it is to feel like you know what you are talking about and have people hang on your words listening so intently!
```
Comment (catlook) [inline _posts/2015-10-21-rackspace-grace-hopper.md]: ... their session, and they both exclaimed ...
... and to have people hang on your words, listening so intently!

File: _posts/2015-10-21-rackspace-grace-hopper.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+Another favorite moment from GHC was hearing from Malini Bhandaru of Intel, and the joint recruiting session we had with Intel. Malini gave a great overview of OpenStack in the open source track, and shared that she had been to the [first Grace Hopper conference in the mid-1990s when it was about 500 women in tech in Washington, DC](https://en.wikipedia.org/wiki/Grace_Hopper_Celebration_of_Women_in_Computing). To think of where we have been and where we are going is exciting. As I gain wisdom and experience and hope to age gracefully as a woman in tech, I think of the past, present, and future converging and smile, knowing that we are making a difference for woman at many career stages.
```
Comment (catlook) [inline _posts/2015-10-21-rackspace-grace-hopper.md]: ... open source track and shared ...

[ ed note: thanks for great article! Hope I can to next year.  In the mean time, use any of my edits you like and saute the rest in olive oil with tarragon or Italian seasoning!  Ping me, if you have questions - Cat ]

**Comment by annegentle** [comment]
This is ready to be published, thanks!

**Comment by smashwilson** [comment]
On it :clock10:

**Comment by smashwilson** [comment]
Okay, on its way out. It'll be live in _(grumble, grumble)_ about 30 minutes.

**Comment by annegentle** [comment]
Thanks @smashwilson! Much obliged :cake:

**Pull request merged**